services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/internal/client/DefaultDynamoDbEnhancedAsyncClient.java

```diff
@@ -88,13 +87,13 @@ public static Builder builder() {
     }
 
     @Override
-    public CompletableFuture<List<TransactGetResultPage>> transactGetItems(TransactGetItemsEnhancedRequest request) {
+    public CompletableFuture<List<DefaultDocument>> transactGetItems(TransactGetItemsEnhancedRequest request) {
```

Comment:
This should be Document, there's no reason for it to care about the implementation.

Reply:
The unfortunate consequence of auto-rename without considering the switch to an interface. Updating all references.